deployment/NuGet/template/GitTools.IssueTrackers/GitTools.IssueTrackers.nuspec

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>GitTools.IssueTrackers</id>
+    <version>[VERSION]</version>
```

Comment:
I use FinalBuilder, just a basic script for all packages (also picks up the right dependencies, etc). Not really in scope to change anything related to this for now.